atomicapp/nulecule/main.py

```diff
@@ -147,10 +147,19 @@ def genanswers(self, dryrun=False, answers_format=None, **kwargs):
         self.answers_format = answers_format or ANSWERS_FILE_SAMPLE_FORMAT
 
         # Check to make sure an answers.conf file doesn't exist already
-        answers_file = os.path.join(os.getcwd(), ANSWERS_FILE)
+        if location is None:
+            answers_file = os.path.join(os.getcwd(), ANSWERS_FILE)
+        else:
+            answers_file = os.path.join(location, ANSWERS_FILE)
+
         if os.path.exists(answers_file):
             raise NuleculeException(
-                "Can't generate answers.conf over existing file")
+                "Can't generate %s over existing file" % answers_file)
```

Comment:
Might be better to have the file at the end after a colon:

`"Can't generate answers. File exists: %s" % answers_file`